modules/creating-microshift-development-environment.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * microshift-install-rpm.adoc
+
+:_content-type: PROCEDURE
+[id="creating-microshift-development-environment"]
+= Creating the MicroShift development environment
+:context: creating-microshift-development-environment
+
+Use the following procedure to set up your MicroShift development environment. 
+
+.Procedure 
+
+. Download {op-system-first} 8.7 or greater from the link:https://developers.redhat.com/products/rhel/download[Download Red Hat Enterprise Linux] page. 
++
+[NOTE]
+====
+{op-system} 9.0 is currently unsupported for MicroShift deployments.
+====
+
+. Create a {op-system} 8 virtual machine with the following settings: 
+.. 2 cores
+.. 4096 MB of RAM
+.. 50 GB of storage 
++
+[NOTE]
+====
+Visual Studio Code might consume up to 2 GB of RAM. For running the IDE on the development virtual machine, it is recommended to allocate at least 6144 MB of RAM in total. 
+====
+
+. Move the ISO image to the `/var/lib/libvirt/images` directory. 
+
+. Enter the following command to create the virtual machine: 
++
+[source,terminal]
+----
+$ sudo dnf install -y libvirt virt-manager virt-viewer libvirt-client qemu-kvm qemu-img
+
+VMNAME="microshift-dev"
+sudo -b bash -c " \
+cd /var/lib/libvirt/images/ && \
+virt-install \
+    --name ${VMNAME} \
+    --vcpus 2 \
+    --memory 4096 \
+    --disk path=./${VMNAME}.qcow2,size=50 \
+    --network network=default,model=virtio \
+    --os-type generic \
+    --events on_reboot=restart \
+    --cdrom ./rhel-8.6-$(uname -i)-boot.iso \
+"
+----
+
+. When prompted by the OS installation wizard, configure a MicroShift administrator user and root password. 
```

Comment:
The actual user name does not matter here. We specified a user for the developer docs to have the dev environments consistent across the team, but we may not even need a user to create a separate administrative user if they don't want to.